.gitignore

```diff
@@ -9,7 +9,7 @@ gemmini_build/
 amx_build/
 
 .ipynb_checkpoints
-.sys_atl_c_cache
+.exo_c_cache
 
```

Comment:
`git clean -fdx` should work. And I'm not very concerned about people checking in their caches because the PR system will prevent them from finally being merged.